packages/ketcher-core/src/domain/serializers/ket/schema.json

```diff
@@ -713,6 +758,270 @@
           }
         }
       }
+    },
+    "monomerTemplate": {
+      "type": "object",
+      "required": ["type", "id"],
+      "properties": {
+        "type": {
+          "const": "monomerTemplate"
+        },
+        "monomerClass": {
+          "type": "string",
+          "enum": [
+            "AminoAcid",
```

Comment:
currently we have PEPTIDE as key word and AminoAcid as as subclass... is it ok that here is no Peptide?